nav2_costmap_2d/include/nav2_costmap_2d/footprint_subscriber.hpp

```diff
@@ -60,10 +60,15 @@ class FootprintSubscriber
     std::vector<geometry_msgs::msg::Point> & footprint,
     rclcpp::Duration & valid_footprint_timeout);
   /**
-   * @brief Returns an oriented robot footprint without timeout
+   * @brief Returns an oriented robot footprint using timeout specified in constructor
```

Comment:
```suggestion
   * @brief Returns an oriented robot footprint using timeout in argument
```

Reply:
But there's no timeout argument here. This method uses FootprintSubscriber's own footprint_timeout_.